src/PrestaShopBundle/Form/Admin/Sell/Product/CreateProductFormType.php

```diff
@@ -0,0 +1,106 @@
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductType;
+use PrestaShop\PrestaShop\Core\Form\FormChoiceAttributeProviderInterface;
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class CreateProductFormType extends TranslatorAwareType
+{
+    /**
+     * @var FormChoiceProviderInterface|FormChoiceAttributeProviderInterface
+     */
+    private $formChoiceProvider;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param FormChoiceProviderInterface|FormChoiceAttributeProviderInterface $formChoiceProvider
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        $formChoiceProvider
+    ) {
+        parent::__construct($translator, $locales);
+        $this->formChoiceProvider = $formChoiceProvider;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('type', ChoiceType::class, [
+                'choices' => $this->formChoiceProvider->getChoices(),
+                'choice_attr' => $this->formChoiceProvider->getChoicesAttributes(),
+                'required' => true,
+                'label' => false,
+                'empty_data' => ProductType::TYPE_STANDARD,
+                'block_prefix' => 'product_type',
+            ])
+            ->add('create', SubmitType::class, [
+                'label' => $this->trans('Create product', 'Admin.Catalog.Help'),
```

Comment:
```suggestion
                'label' => $this->trans('Add new product', 'Admin.Catalog.Feature'),
```

@jolelievre Where does this string appear? Is it this one?
![image](https://user-images.githubusercontent.com/70583503/146736448-14095867-56a3-4524-8d61-cc064e17bee4.png)

If yes, it is "Add new product" and not "Create product". I think there was a mistake when copying the wording from the wireframes to the Figma mockups.

Also, that would be in the Admin.Catalog.Feature domain.

Reply:
Indeed, the mockups were not up to date, this is fixed